drivers/avr/serial.c

```diff
@@ -20,7 +20,7 @@
 
 #ifdef SOFT_SERIAL_PIN
 
-#    ifdef __AVR_ATmega32U4__
+#    if defined(__AVR_ATmega32U4__) || defined(__AVR_ATmega32U2__)
 // if using ATmega32U4 I2C, can not use PD0 and PD1 in soft serial.
 #        ifdef USE_AVR_I2C
 #            if SOFT_SERIAL_PIN == D0 || SOFT_SERIAL_PIN == D1
```

Comment:
The line below this should be changed to `ATmegaxxU4`.

Reply:
Fixed on fdb63faf70ef768193b5207b432641714dda6349